src/test/java/spark/BodyAvailabilityTest.java

```diff
@@ -18,9 +18,11 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(BodyAvailabilityTest.class);
 
     private static final String BODY_CONTENT = "the body content";
-
+    
     private static SparkTestUtil testUtil;
 
+    private final int HTTP_OK = 200;
+    
```

Comment:
Maybe removing this extra line? Also I feel it would be better to group all the constant together?

Reply:
Hello, Ariel!

Let me see if I get this straight... By grouping all the constant together you mean creating some sort of class for all HTTP status or do you only mean stacking up all the constants in lines close to each other?

Reply:
Hi! 

By grouping them together I was referring to just have them one next to another something like: 

```
    private static final Logger LOGGER = LoggerFactory.getLogger(BodyAvailabilityTest.class);
    private final int HTTP_OK = 200;
    private static final String BODY_CONTENT = "the body content";

    private static SparkTestUtil testUtil;


It's a minor thing tho 😅